### ISSUE TITLE
method for collecting system

### ISSUE THREAD

**Issue by anbu-squad** [UnamSanctam/SilentCryptoMiner#2267]
Method for collecting system information uses system calls or API ?

**Comment by UnamSanctam** [comment]
You mean in the miners? Then it uses APIs, I don't believe it's possible to get the system information required using system calls.

**Comment by Silentsniper1** [comment]
> llecting system information uses system calls or API ?

contact me on discord ill help you out username is :Silentsniper0

**Comment by anbu-squad** [comment]
yes, I’m interested in how the code for sending requests is organized in the miners themselves
I still want to do the same
send requests
receive configuration
process responses

you have to somehow put together the puzzle together with artificial intelligence

it would be really nice to see the methods

**Comment by anbu-squad** [comment]
if you take a separate array API
then it turns out that the remote configuration simply updates the array API?

**Comment by UnamSanctam** [comment]
>yes, I’m interested in how the code for sending requests is organized in the miners themselves
I still want to do the same
send requests
receive configuration
process responses
you have to somehow put together the puzzle together with artificial intelligence
it would be really nice to see the methods

It's quite simple using cpp-httplib.

>if you take a separate array API
then it turns out that the remote configuration simply updates the array API?

Yes the "remote-config" and "api-endpoint" options just update the arrays if that's what you're asking.

**Comment by anbu-squad** [comment]
yes, I wanted to make sure that my ideas about how queries work are correct

however, it is still interesting to see the code that is in the miners
If not publicly, is it possible to get non-email?
The chat wrote me part of the code, but I can’t be sure it’s correct, I want to understand it by comparison
